Directory.Build.targets

```diff
@@ -1,7 +1,10 @@
-﻿<Project>
+<Project>
   <PropertyGroup>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(MicrosoftNETCoreApp20PackageVersion)</RuntimeFrameworkVersion>
```

Comment:
remove 2.0 and 2.1 references in this PR?

Reply:
We separated that out into aspnet/Universe#1153.